grails-app/controllers/org/pih/warehouse/api/StockTransferApiController.groovy

```diff
@@ -119,12 +119,8 @@ class StockTransferApiController {
         render([data: stockTransfer?.toJson()] as JSON)
     }
 
-    Boolean isNull(Object objectValue) {
-        return objectValue == JSONObject.NULL || objectValue == null || objectValue?.equals("")
-    }
-
     StockTransfer bindStockTransferData(StockTransfer stockTransfer, User currentUser, Location currentLocation, JSONObject jsonObject) {
-        bindData(stockTransfer, jsonObject)
+        bindData(stockTransfer, jsonObject, [exclude: ['stockTransferItems']])
```

Comment:
comparing both grails1 and grails3 versions I noticed that in grails1 when doing `bindData(stockTransfer, jsonObject)` it would not bind **stockTransferItems** so we had to do it manually.
In grails3 it does initally bind **stockTransferItems**, at least some of the values, but some other values we have to bind ourselves. 
I decided to mimic the grails1 behavior and just excluded **stockTransferItems** from being bound to **stockTransfer** object